src/main/kotlin/no/nav/familie/ef/sak/behandling/BehandlingPåVentService.kt

```diff
@@ -187,6 +217,27 @@ class BehandlingPåVentService(
         }
     }
 
+    private fun validerKanOppretteVurderHenvendelseOppgave(
+        saksbehandling: Saksbehandling,
+        vurderHenvendelseOppgaver: List<VurderHenvendelseOppgavetype>,
+    ) {
+        if (vurderHenvendelseOppgaver.contains(VurderHenvendelseOppgavetype.INFORMERE_OM_SØKT_OVERGANGSSTØNAD)) {
+            brukerfeilHvis(saksbehandling.stønadstype != StønadType.OVERGANGSSTØNAD) {
```

Comment:
brukerFeil, eller feil?

Reply:
Tror vi vil ha `feilHvis` her - for dersom dette skjer så har vi en bug i koden - og brukerfeil gir bare info/warning i loggene.

Reply:
Fixd [ce6fff7](https://github.com/navikt/familie-ef-sak/pull/2184/commits/ce6fff725e7da715baf68b034a915cca65f21ba6)